src/ClickSendApi.php

```diff
@@ -113,16 +105,18 @@ public function sendManySms(array $messages)
             }
             if ($worked == count($messages)) {
                 $result['success'] = true;
-                $result['message'] = 'Messages sent successfully.';
+                $result['message'] = count($messages).' of '.count($messages).' '.strtolower($this->response->response_msg);
```

Comment:
Being picky, but since we know that $worked == count($messages), we could write this "$worked of $worked " instead of two calls to count(). Save the (very few) CPU cycles.

Reply:
Good point! Will pick it up on the next revision - thanks 💪